quran/quran.com-frontend-next
# styles: add isSidebarNavigationVisible check & update the footer styles

### Summary
Fixes #1837

On the Quran reader page, when opening the side navbar we see that the Footer is hidden under the side navbar while the reset of the page adjust to the available width.

### Test Plan

1. Select any Surah
2. Scroll to the bottom of the page until you see the footer
3. Open the side navbar


### Screenshots

| Before | After |
| ------ | ------ |
| ![2022-11-11 22 54 12 quran com d70f3efc3d9b](https://user-images.githubusercontent.com/18232579/201438852-89fbd348-3700-4631-8edf-89f97455a855.png) | ![2022-11-11 23 20 42 localhost bfdb7a858ef2](https://user-images.githubusercontent.com/18232579/201438856-5e31c19b-a30e-4bc4-ab0f-45a4747423ed.png) |
